staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -4233,6 +4233,13 @@ func (d *ConfigMapDescriber) Describe(namespace, name string, describerSettings
 			w.Write(LEVEL_0, "%s:\n----\n", k)
 			w.Write(LEVEL_0, "%s\n", string(v))
 		}
+		w.Write(LEVEL_0, "\nBinaryData\n====\n")
+		for k, v := range configMap.BinaryData {
+			w.Write(LEVEL_0, "%s:\n----\n", k)
+			w.Write(LEVEL_0, "%s bytes\n", strconv.Itoa(len(v)))
```

Comment:
Small opinion here: putting everything on the same line could turn things more readable:
```suggestion
			w.Write(LEVEL_0, "%s: %s bytes\n", k, strconv.Itoa(len(v)))
```

So instead of having 
```
BinaryData
====
1.png:
----
16091 bytes
2.png:
----
16091 bytes
3.png:
----
16091 bytes
```

Will have:

```
BinaryData
====
1.png: 16091 bytes
2.png: 16091 bytes
3.png: 16091 bytes

```

Reply:
@rikatz I was think about the same thing too, however, for the `Data` part, it was represent as 
```
Data
====
key1:
----
value1
key2:
----
value2
```

should we be consistent or should we change the way to represent the Data? I prefer your way but was also thinking of consistency

Reply:
hum, consistency with what? I guess Data and BinaryData have different objectives, and as the BinaryData does not exists in description anyway, MY OPINION is that we can create the way we think is more clear for the user :)

Because no value other then a default of size, in bytes, is shown, we can say it will keep consistency with formatting across multiple BinaryData values, which for me, as a user seems more clear (and parseable, as well, if I want to grep things!) :)

Reply:
@rikatz  consistent of how you represent the Key Value pair for Data and Binary data and yea, will change that when I get some time